### PR TITLE
6 minimal beam model

### LINKED ISSUE #6: add a minimal beam model
After discussing with @grzanka. Fitting will be more robust if there is some initial width of the beam.

Propose:
- 4 mm FWHM for all beams
- 60 cm exit-window to isocenter distance
- 2 meters from last quadrupole to isocenter
- add air as material
- add exit window material
- add energy spread 1 %
- add divergence 3 mrad

### PR DESCRIPTION
**Pull request by nbassler**
Closes #6.
- [x] 200 cm quadrupole <-> phantom surface distance
- [x] beam pipe vacuum -300 cm -> -150 cm. Then air.
- [x] 4 mm inital FWHM for all beam, set at quadrupole (@ -300 cm)
- [x] 150 cm exit-window to phantom surface distance
- [x] 60 cm RIFI to phantom surface distance
- [x] add air as material
- [x] add exit window material (25 um Titanium)
- [x] add energy spread 1 %
- [x] add divergence 2.5 mrad

```
<--------150 cm-------><--------150 cm----------><---- 50 cm ------->
======================|              ||          |.....target...... |
                                                 |
beam pipe                    air   rifi(pmma)    |  @ 0.0 cm  water  

```

### CODE REVIEW

**Comment by nbassler** [comment]
10.000 primaries, 20Ne, 3mm Rifi.

dcyl
ecylrie
emshs

**Comment by nbassler** [comment]
possibly 200.000 primaries may be enough.

File: prepare.py [hunk 1]
```diff
@@ -5,6 +5,20 @@
...
 
+BEAM_DIV = 2.5        # beam divergence in mrad
+BEAM_FOCUS = 0.0      # beam focus relative to beam starting position, positive if upstream (defocussed)
+ENERGY_SPREAD = 0.01  # relative energy spread 0.01 = 1 %.
+
+# ripple filter material ID, change according to mat.dat, and AIR if no RIFI
+MAT_RIFI = 4          # PMMA
+MAT_NORIFI = 2        # AIR
+
+# Zone number oif RIFI, as specified in geo.dat
```
Comment (grzanka) [inline prepare.py:16]: typo here `oif`

**Comment by grzanka** [comment]
Nice job !

**Review by grzanka — APPROVED** [review]

**Comment by fkalholm** [comment]
Very nice job! I have checked everything and I get the general idea/structure of it. But it will probably take a year or so until I can do these things myself!

**Review by fkalholm — APPROVED** [review]

File: prepare.py (hunk 1 above)
Comment (nbassler) [reply prepare.py:16]: will fix in different branch

**Pull request merged**